test/integration/targets/filters/runme.sh

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env bash
 
-set -ex
-virtualenv --system-site-packages --python "${ANSIBLE_TEST_PYTHON_INTERPRETER}" "${OUTPUT_DIR}/venv"
-source "${OUTPUT_DIR}/venv/bin/activate"
 set -eux
```

Comment:
This is not really related to virtualenv change. But I think we should promote safer mode in scripts:
```suggestion
 set -euxo pipefail
```

Reply:
That's probably something worth changing across all of the tests at once.

None of these tests use pipes, so there's no need to change it in this PR.